### PR TITLE
fix d1 list retrieval (needed for the deploy)

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
by:
 - updating the wrangler version (so that the listing json format is available)
 - updating the retrieval d1 id command
 
@xtuc I've tested the command locally and it seems to work nicely, I am not sure how to test it in the deploy action... I guess we need to merge this in order to see if it works?

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@ jobs:
       - name: retrieve D1 database
         uses: cloudflare/wrangler-action@2.0.0
         with:
-          command: d1 list | grep "wildebeest-${{ env.NAME_SUFFIX }}\s" | awk '{print "d1_id="$2}' >> $GITHUB_ENV
+          command: d1 list --json | jq -r '.[] | select(.name == "wildebeest-cloudflare") | .uuid'
```
Comment (xtuc) [inline .github/workflows/deploy.yml:100]:
```suggestion
          command: d1 list --json | jq -r '.[] | select(.name == "wildebeest-${{ env.NAME_SUFFIX }}") | .uuid'
```

Comment (dario-piotrowicz) [reply .github/workflows/deploy.yml]: silly me! sorry about that 😓

**Review by xtuc — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -97,7 +97,7 @@ jobs:
       - name: retrieve D1 database
         uses: cloudflare/wrangler-action@2.0.0
         with:
-          command: d1 list | grep "wildebeest-${{ env.NAME_SUFFIX }}\s" | awk '{print "d1_id="$2}' >> $GITHUB_ENV
+          command: d1 list --json | jq -r '.[] | select(.name == "${{ env.NAME_SUFFIX }}") | .uuid' | awk '{print "d1_id="$2}' >> $GITHUB_ENV
```
Comment (xtuc) [inline .github/workflows/deploy.yml:100]:
```suggestion
          command: d1 list --json | jq -r '.[] | select(.name == "${{ env.NAME_SUFFIX }}") | .uuid' | awk '{print "d1_id="$1}' >> $GITHUB_ENV
```

**Comment by dario-piotrowicz** [comment]
Something's wrong with the new wrangler version and we need to fix the backend first 😓

**Pull request closed without merging**